Gemfile

```diff
@@ -0,0 +1 @@
+"rubocop"
```

Comment:
I think this needs to be `gem "rubocop"` and have a `source "https://rubygems.org"` line. Otherwise `bundle install` doesn't do anything:
```
➜  tsuru-performance-tests git:(98091996-deploy-mulitple-apps-tsuru) ✗ bundle
The Gemfile specifies no dependencies
```